lib/from_erlang.ml

```diff
@@ -44,9 +44,36 @@ let rec expr_of_erlang_expr = function
      raise Known_error.(FialyzerError (NotImplemented (make_issue ~url:"https://github.com/dwango/fialyzer/issues/79")))
   | ExprRemoteFunRef (_line_t, m, f, a) ->
     MFA {module_name = expr_of_atom_or_var m;  function_name = expr_of_atom_or_var f; arity = expr_of_integer_or_var a}
-  | ExprFun (_line_t, name, clauses) ->
-     (* TODO: support `fun (...) -> ... end` *)
-     raise Known_error.(FialyzerError (NotImplemented (make_issue ~url:"https://github.com/dwango/fialyzer/issues/80")))
+  | ExprFun (_line_t, name_option, clauses) ->
+    let name = Option.value_map name_option ~default:(Variable.create()) ~f:(fun x -> x) in
+    (* Create a list which have n elements *)
+    let rec fill e = (function
+    | 0 -> []
+    | n -> (e())::(fill e (n - 1))
+    )
+    in
+    let (cs, arities) = clauses |> List.map ~f:(function
+    | F.ClsCase (_, _, _, _) -> 
+      failwith "cannot reach here"
+      (* Ignore guards currently since guard is complex and it's not needed for simple examples *)
+    | F.ClsFun (_, patterns, _, body) ->
+      let rec pattern_of_mini_erlang = function
+      | F.PatVar(_, v) -> PatVar v
+      | F.PatUniversal _ -> PatVar "_"
+      | F.PatTuple(_, patterns) -> PatTuple (patterns |> List.map ~f:pattern_of_mini_erlang)
+      | _ -> failwith "not supported yet. mini-Erlang cannot support all Erlang pattern currently"
```

Comment:
Please make a issue and replace this `failwith` to `Known_error.(FialyzerError (NotImplemented issue))`

Reply:
I see.